### DISCUSSION TITLE
Infinite customizability and SQL queries

### DISCUSSION THREAD

**Discussion by punchagan** [blr-today/community#2] [General]
@captn3m0 thanks for the latest version of this project! I am quite excited about this idea. 

I noticed on the /about page that the project eventually would allow folks to run their own SQL queries. I was curious to know if you had ideas on how to allow this with a static site. Are you considering in-browser SQLite stuff? Or do you have other ideas?

Also, I was curious how this might tie in with the the thesis - "The best UX for event discovery is your existing calendar app."

**Comment by captn3m0** [comment]
Planning to run a tiny server less thing  elsewhere, probably on Netlify Functions. In-browser won't work, since the response will be rendered as ICS and is fetched by calendar apps, which are no-js user agents. Similar to how you can't generate RSS feeds using browser JS.

**Reply by punchagan** [reply]
Makes sense! Thanks!
